### ISSUE TITLE
TransactionDetails class has typo on fee parameter

### ISSUE THREAD

**Issue by thunderbiscuit** [bitcoindevkit/bdk-ffi#136]
The Rust struct has a field called `fee` ([docs](https://docs.rs/bdk/latest/bdk/struct.TransactionDetails.html)) whereas our resulting class uses `fees`:

```kotlin
data class TransactionDetails (
    var fees: ULong?, 
    var received: ULong, 
    var sent: ULong, 
    var txid: String 
) {
    // ...
}
```

Just noticed this while building the API docs for bdk-kotlin.

**Comment by dhruv-1001** [comment]
I'll take this one.

**Comment by thunderbiscuit** [comment]
Sounds good. Building and testing bdk-ffi is not always easy, let me know if you have trouble setting yourself up.

Also maybe wait just a few days before spending too much energy on this, I want to make sure this was not intentional in any way and that fixing it is a good idea.

**Comment by dhruv-1001** [comment]
Thanks! I was able to setup the project.
Okay, I'll wait some time before fixing this issue.

**Comment by thunderbiscuit** [comment]
All right @dhruv-1001 go ahead with this!

**Comment by dhruv-1001** [comment]
Okay cool.

**Comment by thunderbiscuit** [comment]
This was fixed with #145.
